quad/src/qk.rs

```diff
@@ -8,33 +8,26 @@ pub fn qk_quadrature<const M: usize, F>(
     xgk: &[f64; M],
     wgk: &[f64],
     wg: &[f64],
-) -> (Vec<f64>, f64, f64)
+) -> (ndarray::Array1<f64>, f64, f64)
 where
-    F: Fn(f64) -> Vec<f64>,
+    F: Fn(f64) -> ndarray::Array1<f64>,
```

Comment:
Here and in the other `qkXY.rs`: in order to shorten a bit the function signature, import `Array1` in the namespace:

```rs
use ndarray::Array1
```